meet-data/uspa/0799/meet.csv

```diff
@@ -1,2 +0,0 @@
-Federation,Date,MeetCountry,MeetState,MeetTown,MeetName
-USPA,2017-03-18,USA,MO,O'Fallon,3rd Powerbody USA Open
```

Comment:
The existing meet file was correctly numbered as per http://uspa.net/competition_results.html.